brands/shop/clothes.json

```diff
@@ -344,6 +344,18 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|BonWorth": {
+    "countryCodes": ["us"],
+    "nomatch": ["shop/clothes|BonWorth"],
```

Comment:
Can you please remove this `nomatch` line - it matches this entry!

The "nomatch" property is used to suppress duplicate warnings.  It would be used if there were a different entry with the same name and we need to let the matcher know that they are two different things.  (For example a gas station brand also named "BonWorth").

Reply:
Sorry about that, I'll get it sorted!